### PR TITLE
Add configurable static ip address to the image

### PR DESCRIPTION
**Pull request by BassemMohsen**
Configuring STATIC_IP_ADDRESS fixes the issue with poky ssh tests.

### CODE REVIEW

File: recipes-core/systemd/systemd_%.bbappend [hunk 1]
```diff
@@ -7,5 +7,9 @@ SRC_URI += "file://10-pelux-ethernet.network"
 # We also make sure resolv.conf is using systemd-resolved
 do_install_append() {
     install -m 0644 ${WORKDIR}/10-pelux-ethernet.network ${D}${sysconfdir}/systemd/network/10-pelux-ethernet.network
+    if [ ! -z "${STATIC_IP_ADDRESS}" ]; then
+    	echo '[Address]' >> ${D}${sysconfdir}/systemd/network/10-pelux-ethernet.network
+    	echo "Address=${STATIC_IP_ADDRESS}/24" >> ${D}${sysconfdir}/systemd/network/10-pelux-ethernet.network
```
Comment (tobsan) [inline recipes-core/systemd/systemd_%.bbappend]: Make up your mind about single quotes versus double quotes.

File: conf/variant/common/local.conf [hunk 2]
```diff
@@ -4,6 +4,9 @@ DL_DIR = "${TOPDIR}/downloads"
 SDKMACHINE = "x86_64"
 DISTRO = "pelux"
 
+# Target Static IP address
```
Comment (tobsan) [inline conf/variant/common/local.conf]: A better comment would be nice here. Why is this a good thing, what is it used for etc.

Comment (BassemMohsen) [reply conf/variant/common/local.conf]: I added the comment now.

File: recipes-core/systemd/systemd_%.bbappend (hunk 1 above)
Comment (BassemMohsen) [reply recipes-core/systemd/systemd_%.bbappend]: double quotes usage is unified.

**Pull request merged**